### ISSUE TITLE
Chinese Home Page Translation (i18n)

### ISSUE THREAD

**Issue by JonSEng** [International-Mentorship-ISMP/ismp#15]
https://trello.com/c/lD3GlF6H/15-home-page-secondary-page-translation-i18n
 
- [x] Actually translate (i.e., we only implemented the ability to **choose** a translation)

- [x] Chinese homepage translation

- [x] Chinese string translation

- [x] Fill in the home page paragraphs with the correct translation

- [x] Collect any other strings you need translated and send them to @lillyglin or @douglaschan32167 

Translations of the old paragraphs on the home page:
https://drive.google.com/file/d/1vkpHikPQnBvIQtPm5HbjCa2k-saJrT-N/view

Some other strings (including what goes on the footer)
https://docs.google.com/document/d/1ssRT47aA0BVtNZLXf0IzQw3khM4plsA6JzdD6xiY3EM/edit

**Comment by richyen** [comment]
TODO: Actually translate (i.e., we only implemented the ability to **choose** a translation)

**Comment by jessewoo** [comment]
@richyen - Implemented the ability, need translators.  This need to be assign to content people, should create a new ticket.

**Comment by douglaschan32167** [comment]
I asked Pacie to translate the home page to Japanese and Tiffany Wu to translate it to Chinese. Moving this ticket to blocked until we get the actual translations.

**Comment by JonSEng** [comment]
Hi @lillyglin what's the "Secondary Page"?

Also FYI Home Page translations of content here https://3.basecamp.com/4474129/buckets/16925967/vaults/2774213712

**Comment by lillyglin** [comment]
@JonSEng I don't think we have a secondary page right now...it's probably for all the inner subpages for v2.

Also, let's make a new issue for Korean translation. Waiting on the Korean web strings.

**Comment by lillyglin** [comment]
@JonSEng Moving this over to selected for development.

**Comment by douglaschan32167** [comment]
Translations of the old paragraphs on the home page:
https://drive.google.com/file/d/1vkpHikPQnBvIQtPm5HbjCa2k-saJrT-N/view

Some other strings (including what goes on the footer)
https://docs.google.com/document/d/1ssRT47aA0BVtNZLXf0IzQw3khM4plsA6JzdD6xiY3EM/edit

**Comment by JonSEng** [comment]
<img width="1818" alt="Screenshot 2020-06-27 10 15 02" src="https://user-images.githubusercontent.com/1644269/85928064-190e6880-b85f-11ea-90f1-e1ce27daa701.png">

We do need to do the footer.
We don't need to translate testimonials and blog posts.
<img width="1876" alt="Screenshot 2020-06-27 10 15 10" src="https://user-images.githubusercontent.com/1644269/85928065-1b70c280-b85f-11ea-803c-a85d463ae1da.png">

**Comment by JonSEng** [comment]
https://3.basecamp.com/4474129/buckets/16925967/todos/2797902626 for getting missing strings - @lillyglin is on it!

**Comment by lillyglin** [comment]
@kailanjian Talking to translation team. They have the latest edits on the English verbiage. I'll poke to see if they can get it done today and up.

**Comment by lillyglin** [comment]
Closing issue, since this has already been updated.
